examples/backends/vllm/launch/lora/multimodal/lora_agg.sh

```
@@ -14,7 +14,10 @@
 #   ./lora_agg.sh -- --enforce-eager                          # Pass extra args to vLLM
 
 set -euo pipefail
-trap 'echo "Cleaning up..."; kill 0' EXIT
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)